### PR TITLE
i18n: update KO 7.0 translations

### PR DESCRIPTION
**Pull request by Bing-su**
This is the 3rd of PRs for the Korean client 7.0, which will be updated on December 3rd.

Maintenance: [2024-12-02 12:00](https://www.timeanddate.com/worldclock/converter.html?iso=20241202T030000&p1=235&p2=137&p3=64&p4=263&p5=195&p6=37&p7=33&p8=248) ~ [2024-12-03 18:00](https://www.timeanddate.com/worldclock/converter.html?iso=20241203T090000&p1=235&p2=137&p3=64&p4=263&p5=195&p6=37&p7=33&p8=248) (KST, 30 hours) 

- 1st https://github.com/OverlayPlugin/cactbot/pull/523
- 2nd https://github.com/OverlayPlugin/cactbot/pull/524
- 3rd (Here) https://github.com/OverlayPlugin/cactbot/pull/525

---

- raidboss 7.0 dungeons, trials, hunts.

### CODE REVIEW

**Comment by Bing-su** [comment]
Question

https://github.com/OverlayPlugin/cactbot/blob/070a02d339c1d8886d12fa38792f72e1ad810edc/ui/raidboss/data/07-dt/trial/valigarmanda-ex.ts#L992

`?` (question mark) is also a regular expression, so does it need to be escaped?

**Comment by Akurosia** [comment]
> Question
> 
> https://github.com/OverlayPlugin/cactbot/blob/070a02d339c1d8886d12fa38792f72e1ad810edc/ui/raidboss/data/07-dt/trial/valigarmanda-ex.ts#L992
> 
> `?` (question mark) is also a regular expression, so does it need to be escaped?

Yes you are right, it also needs to be escaped, good catch

File: ui/raidboss/data/07-dt/trial/valigarmanda.ts [hunk 1]
```diff
@@ -556,6 +556,41 @@ const triggerSet: TriggerSet<Data> = {
...
+        'Flame-kissed Beacon': '불의 상징',
+        'Valigarmanda': '발리가르만다',
+      },
+      'replaceText': {
+        '\\(AoEs\\)': '(장판)',
+        '\\(ice phase\\)': '(얼음 페이즈)',
+        '\\(knockback\\)': '(넉백)',
+        '\\(storm phase\\)': '(번개 페이즈)',
+        '\\(tower\\)': '(기둥)',
+        '\\(storm/ice phase\\?\\)': '(번개/얼음 페이즈?)',
```
Comment (valarnin) [inline ui/raidboss/data/07-dt/trial/valigarmanda.ts:572]: (not directly related to this PR)

Other languages seem to be missing this replacement. Is it not showing in the "find missing timeline translations" tool, or was it just not translated?

**Review by valarnin — APPROVED** [review]
I don't usually review translation PRs, so please take this approval in the vein of "this looks technically correct from a code standpoint".

File: ui/raidboss/data/07-dt/trial/valigarmanda.ts (hunk 1 above)
Comment (Bing-su) [reply ui/raidboss/data/07-dt/trial/valigarmanda.ts:572]: https://github.com/OverlayPlugin/cactbot/blob/3ebdd7a9612c9de75f40ba20e87f56c585f3853c/ui/raidboss/data/07-dt/trial/valigarmanda.txt#L101

Parenthesized phrases that are not present in .ts files, but only in .txt files, are not checked by the find-translations tool, and do not seem to be treated as missing translations.

**Pull request merged**